docs/DEPLOYMENT.md

```diff
@@ -294,18 +294,54 @@ Update your GitHub OAuth created with the UI route in place of `Homepage URL` an
 
 ## Add resources in DB
 
-1. Add your GitHub username with required scopes in [config.yaml][config-yaml]. For adding resources in DB, you will need   `catalog:refresh` scope.
+1. Add your GitHub username with required scopes in [config.yaml][config-yaml]. For adding resources in DB, you will need `catalog:refresh` scope.
```

Comment:
should we ask the user to push the updated config.yaml to their respected branch and update the configmap with the new location of config.yaml if in case they want to deploy the hub on their own cluster?